blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -54,8 +54,12 @@ export class PipelinePage extends Component {
         const { location = {} } = this.context;
 
         const { organization, name, fullName, fullDisplayName } = pipeline || {};
-        const orgUrl = buildOrganizationUrl(organization);
-        const activityUrl = buildPipelineUrl(organization, fullName, 'activity');
+        
+        const organizationName = organization || AppConfig.getOrganizationName();
+        const organizationDisplayName = organization === AppConfig.getOrganizationName() ? AppConfig.getOrganizationDisplayName() : organization;
```

Comment:
It shouldn't be false because we support only one organization, but given that the organization comes from the URL and not from the config from the preloaded I thought it would be safe to add the check. It didn't seem right to directly use the organization from the url and display name from the config without even a check.